stdlib/experimental/query/from.flux

```diff
@@ -24,3 +26,30 @@ inBucket = (
         |> filterMeasurement(measurement)
         |> filter(fn: predicate)
         |> filterFields(fields)
+
+splitData = (v, t=",") => strings.split(v:v, t:t)
+sanitizeList = (v) => strings.replaceAll(v:v, t: ", ", u: ",")
+
+select = (
+    src,
```

Comment:
Do you think this might be easier if the `src` and `fields` were simplified? I am curious if it would be better to do something like this and split it up:

```
import "influxdata/influxdb"

from = (bucket, start, stop=now()) => influxdb.from(bucket) |> range(start, stop)
select = (tables=<-, m, f) => tables |> filter(fn: (r) => r._measurement == m and r._field == f)
```

This way the query would be simplified to:

```
query.from(bucket: "telegraf", start: -5m)
    |> query.select(m: "cpu", f: "usage_user")
```

If successful, we can make the `from()` a permanent change to `from()`. It is something I have been thinking about.

Another thing that you can do is:

```
select = (tables=<-, m, f="") => {
    s = tables |> filter(fn: (r) => r._measurement == m)
    return if f != "" then s |> filter(fn: (r) => r._field == f) else s
}
```